pom.xml

```diff
@@ -79,6 +79,11 @@
             <artifactId>workflow-step-api</artifactId>
             <version>2.13</version>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.marshalling</groupId>
+            <artifactId>jboss-marshalling-river</artifactId>
+            <version>1.4.12.jenkins-3</version> <!-- https://github.com/jglick/jboss-marshalling/compare/1.4.12.Final...1.4.12.jenkins-3 -->
```

Comment:
FTR this is likely due to IDEA not grokking [this](https://github.com/jenkinsci/workflow-cps-plugin/blob/b3ab42c708d832c658c0f1d9c4935e89f2c5a147/.mvn/maven.config#L1).